app/client/src/components/pages/Community.Tabs.Restore.js

```diff
@@ -399,6 +430,84 @@ function Restore() {
             </TabPanel>
+
+            <TabPanel>
+              <>
+                <p>Some descriptive text</p>
+                {grtsStories.status === 'fetching' && <LoadingSpinner />}
+                {grtsStories.status === 'failure' && (
+                  <div css={errorBoxStyles}>
+                    <p>{restoreStoriesError}</p>
+                  </div>
+                )}
+                {grtsStories.status === 'success' && (
+                  <>
+                    {sortedStoriesData.length === 0 && (
+                      <div css={infoBoxStyles}>
+                        <p css={textStyles}>
+                          There are no stories available for the{' '}
+                          <em>{watershed.name}</em> watershed.
+                        </p>
+                      </div>
+                    )}
+
+                    {sortedStoriesData.length > 0 && (
+                      <AccordionList
+                        title={
+                          <>
+                            There{' '}
+                            {sortedStoriesData.length === 1 ? 'is' : 'are'}{' '}
+                            <strong>
+                              {sortedStoriesData.length.toLocaleString()}
+                            </strong>{' '}
+                            {sortedStoriesData.length === 1
+                              ? 'story'
+                              : 'stories'}{' '}
+                            about Nonpoint Source projects funded from EPA
+                            grants under the{' '}
+                            <GlossaryTerm term="Clean Water Act Section 319 Projects">
+                              Clean Water Act Section 319
+                            </GlossaryTerm>{' '}
+                            that benefit waterbodies in the{' '}
+                            <em>{watershed.name}</em> watershed.
+                          </>
+                        }
+                      >
+                        {sortedStoriesData.map((item) => (
+                          <AccordionItem
+                            ariaLabel={item.ss_title}
+                            key={item.ss_seq}
+                            title={<strong>{item.ss_title}</strong>}
+                          >
+                            <div css={storyStyles}>
+                              <p>{item.ss_overview}</p>
+                              <div css={linkBoxStyles}>
+                                <a
+                                  href={item.web_link}
+                                  target="_blank"
+                                  rel="noopener noreferrer"
+                                >
+                                  <i
+                                    css={iconStyles}
+                                    className="fas fa-file-alt"
+                                    aria-hidden="true"
+                                  />
+                                  {getExtensionFromPath(item.web_link)}
```

Comment:
```suggestion
                                  Full Story (
                                  {getExtensionFromPath(item.web_link)})
```

I suggest changing the link text to `Full Story (PDF)`, so that it's more clear what the link will open.

Reply:
I like that, good call.